frontend/src/components/file-explorer/FileExplorer.tsx

```diff
@@ -167,17 +167,19 @@ function FileExplorer() {
           isHidden ? "min-w-[48px]" : "min-w-[228px]",
         )}
       >
-        <div className="flex p-2 items-center justify-between relative">
+        <div className="flex flex-col p-2 relative">
+          <div className="flex items-center justify-end mb-6">
```

Comment:
```suggestion
          <div className="flex items-center justify-end mb-8">
```

Reply:
that's true. thank you for the screenshots and the update. :)